### PR TITLE
finished cookie banner

### PR DESCRIPTION
**Pull request by itay1313**
DX-124

### CODE REVIEW

File: _includes/cookie-consent.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    font-size: 14px;
+    line-height: 22px;
+  }
+  #cookie-notice p {
+    margin: 0;
+  }
+  #cookie-notice a {
+    display: inline-block;
+    cursor: pointer;
+    margin-left: 0.5rem;
```
Comment (j9t) [inline _includes/cookie-consent.html:30]: Please omit leading “0”

File: _includes/cookie-consent.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    margin-right: 5px;
+    cursor: pointer;
+    text-align: center;
+    white-space: nowrap;
+    outline: none;
+    padding: 12px 40px;
+    gap: 2px;
+    border: 1px solid var(--body-text);
+    border-radius: 24px;
+    font-family: "Space Grotesk";
```
Comment (j9t) [inline _includes/cookie-consent.html:45]: Please add generic fallback

File: _includes/cookie-consent.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    color: var(--body-text);
+  }
+  #cookie-notice a.btn:hover {
+    background-color: var(--post-link-hover);
+    color: var(--button-text);
+    border: 1px solid var(--post-link-hover);
+  }
+</style>
+
+<div id="cookie-notice">
```
Comment (j9t) [inline _includes/cookie-consent.html:60]: Comment: What is the source of this text (and this whole banner code)?

File: _includes/google_analytics.html [hunk 4]
```diff
@@ -1,13 +1,9 @@
 <!-- the google_analytics_id gets auto inserted from the config file -->
 {% if site.google_analytics %}
 <!-- Global site tag (gtag.js) - Google Analytics -->
-<script src="https://www.googletagmanager.com/gtag/js?id=UA-277946-54" async></script>
-<script>
```
Comment (j9t) [inline _includes/google_analytics.html:5]: Can you share more, what happened here? This does look suspicious indeed (duplicate UA IDs)—are we just removing it?

**Review by j9t — CHANGES_REQUESTED** [review]
(This is urgent and important, but no expectation for you to handle this this week.)

**Comment by itay1313** [comment]
@j9t Hey Please checkout, have a great week :)

File: _includes/cookie-notice.html [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+  }
+</style>
+
+<div id="cookie-notice">
+  <p>
+    This website uses cookies to ensure you get the best browsing experience. By
+    continuing to use this website, you consent to our use of these cookies.
+    This website contains proprietary content that belongs to LivePerson and
+    that is intended for educational use by our customers and prospects. Your
+    use of this site is subject to our Terms of Use, which include restrictions
```
Comment (j9t) [inline _includes/cookie-notice.html:71]: I know it’s missing on the screen but please link “Terms of Use” (→ https://www.liveperson.com/policies/terms-of-use/).

**Review by j9t — CHANGES_REQUESTED** [review]
Can you make this available on Staging? (Let me know if this is more complicated, but this way we could test and ask for review first.)

File: _includes/cookie-notice.html [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+      display: none;
+    }
+  }
+</style>
+
+<div id="cookie-notice">
+  <p>
+    This website uses cookies to ensure you get the best browsing experience. By
+    continuing to use this website, you consent to our use of these cookies.
+    <span
```
Comment (j9t) [inline _includes/cookie-notice.html:72]: This and other locations: Weird inner-tag breaks—is this a tool setting?

Not concerned about content breaks until we agreed on a standard, but let’s not break within tags.

**Review by j9t — CHANGES_REQUESTED** [review]

File: _includes/cookie-notice.html [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+      >This website contains proprietary content that belongs to LivePerson and
+      that is intended for educational use by our customers and prospects. Your
+      use of this site is subject to our Terms of Use, which include
```
Comment (j9t) [inline _includes/cookie-notice.html:75]: I’m missing a review comment here—please add a link to our TOU (→ https://www.liveperson.com/policies/apitou/).

**Review by j9t — CHANGES_REQUESTED** [review]

File: _includes/google_analytics.html (hunk 4 above)
Comment (j9t) [reply _includes/google_analytics.html:5]: Please do share :)

File: _includes/google_analytics.html [hunk 8]
```diff
@@ -1,13 +1,9 @@
...
-	window.dataLayer = window.dataLayer || [];
-	function gtag() {
-		dataLayer.push(arguments);
-	}
-	gtag('js', new Date());
-	gtag('config', 'UA-277946-54');
-</script>
-{% endif %}
\ No newline at end of file
+<script
```
Comment (j9t) [inline _includes/google_analytics.html:4]: This file: Please adjust the formatting (no inner-tag breaks).

**Review by j9t — CHANGES_REQUESTED** [review]

**Review by j9t — APPROVED** [review]

**Pull request merged**